src/PluginNodes/Boiler2PluginNodes.cs

```diff
@@ -228,31 +232,36 @@ private ServiceResult SwitchOnCall(ISystemContext context, MethodState method, I
     private void AddEvents()

```

Comment:
Should we call this InitEvents or InitializeEvents?

Reply:
sure. Changed